lib/services/transaction_service.dart

```diff
@@ -13,11 +13,11 @@ class TransactionService {
 
   Future<List<Transaction>?> getTransactions() async {
     try {
-      const String accountId = '817b55aa12212e748e8cc2af91544ea2kcom';
+      const String accountId = 'solarisKontistAccountId';
```

Comment:
Same as above comment - should be marked somehow to know this is temporary + write in a TD item to come back and update it.